quartz/depgraph.ts

```diff
@@ -92,6 +106,15 @@ export default class DepGraph<T> {
 
   // DEPENDENCY ALGORITHMS
 
+  // Add all nodes and edges from other graph to this graph
+  addGraph(other: DepGraph<T>): void {
```

Comment:
should be called `mergeGraph`

Reply:
Changed to mergeGraph, will track og image in a separate PR